frontend/src/data/variables/BrfssProvider.ts

```diff
@@ -29,7 +39,48 @@ class BrfssProvider extends VariableProvider {
     df = this.filterByGeo(df, breakdowns);
     df = this.renameGeoColumns(df, breakdowns);
 
+    // TODO How to handle territories?
+    let acsBreakdowns = breakdowns.copy();
+    acsBreakdowns.time = false;
+
+    let consumedDatasetIds = ["brfss"];
+
     if (breakdowns.geography === "national") {
+      // Because we add together the estimated state totals
+      // we need to get the acs state breakdown
+      acsBreakdowns.geography = "state";
+
+      const acsQueryResponse = await this.acsProvider.getData(
+        new MetricQuery(["population", "population_pct"], acsBreakdowns)
+      );
+      consumedDatasetIds = consumedDatasetIds.concat(
+        acsQueryResponse.consumedDatasetIds
+      );
+
+      const acsPopulation = new DataFrame(acsQueryResponse.data);
+
+      df = joinOnCols(
+        df,
+        acsPopulation,
+        ["fips", "race_and_ethnicity"],
+        "left"
+      );
+
+      df = df.generateSeries({
+        estimated_total_diabetes: (row) =>
+          estimateTotal(
+            row.diabetes_count,
+            row.diabetes_count + row.diabetes_no,
+            row.population
+          ),
+        estimated_total_copd: (row) =>
+          estimateTotal(
+            row.copd_count,
+            row.copd_count + row.copd_no,
+            row.population
+          ),
+      });
+
       df = df
         .pivot(breakdowns.demographicBreakdowns.race_and_ethnicity.columnName, {
```

Comment:
It's a bit odd how breakdowns.getSoleDemographicBreakdown().columnName is used in some places whereas breakdowns.demographicBreakdowns.race_and_ethnicity.columnName is used in others. Can you standardize this usage by defining a breakdownColumnName variable near the start of the function which is set to breakdowns.getSoleDemographicBreakdown().columnName, and using that throughout? That's what we do in other providers